boards/common/arduino-atmega/include/board_common.h

```diff
@@ -47,14 +49,35 @@ extern "C" {
 #define LED0_MASK           (1 << DDB5)
 #endif
 
+#ifdef CPU_ATMEGA32U4
+#define LED0_PIN            GPIO_PIN(2, 7) /*< BUILTIN LED */
```

Comment:
The doxygen comment should be `/**< ... */`, here and below

Reply:
Done